nuxeo-drive-elasticsearch/src/test/java/org/nuxeo/drive/elasticsearch/TestESSyncRootFolderItem.java

```diff
@@ -59,8 +58,8 @@
  */
 @RunWith(FeaturesRunner.class)
 @Features({ NuxeoDriveFeature.class, RepositoryElasticSearchFeature.class })
-@Deploy("org.nuxeo.drive.elasticsearch:OSGI-INF/nuxeodrive-elasticsearch-adapter-contrib.xml")
-@LocalDeploy("org.nuxeo.drive.elasticsearch:OSGI-INF/test-nuxeodrive-elasticsearch-contrib.xml")
+@Deploy({"org.nuxeo.elasticsearch.core", "org.nuxeo.elasticsearch.core.test:elasticsearch-test-contrib.xml",
```

Comment:
`"org.nuxeo.elasticsearch.core"` is probably not needed since it is included by the `RepositoryElasticSearchFeature`

Reply:
you are right I will try to remove them